lambda_functions/user_service/get_user.py

```diff
@@ -24,7 +24,21 @@ def get_user(event, context, is_test=False, session=None):
             user_dict = user.to_dict()
             progress = user_handler.get_user_progress(
                 user, is_test, session)
+            total_rank = user_handler.get_user_rank(
+                user, False, is_test, session)
+            level_rank = user_handler.get_user_rank(
+                user, True, is_test, session)
+            solved_cases_total = user_handler.get_solved_cases(
+                user, False, is_test, session)
+            solved_cases_today = user_handler.get_solved_cases(
+                user, True, is_test, session)
+            exp_needed = user_handler.get_needed_exp(user, is_test, session)
```

Comment:
Wouldn't it be possible to get all data in one DB query?

Reply:
Maybe, but I wouldnt really see the point as this query would be incredibly complicated. We could combine the methods and return one dict, but that would make the methods less reusable.

Reply:
If we want to reduce the number of queries, I would rather create a view that extends the user table. But for now I would leave it as it is.